### PR TITLE
Add the ability to define useragents

### PR DESCRIPTION
**Pull request by baselakasha**
I have implemented a way to specify a user agent for the action to use. This allows the user to whitelist this action if they are using some kind of bot firewall.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,13 +1,27 @@
 const core = require('@actions/core');
 const github = require('@actions/github');
 const https = require('https');
+const url = require('url');
 
 try {
   const urlToHit = core.getInput('url-to-hit');
+  const parsed_url = url.parse(urlToHit);
```
Comment (srt32) [inline index.js]: Mind using camelCase?

**Review by srt32 — COMMENTED** [review]
One comment about style but otherwise I'd be happy to merge this!

File: index.js (hunk 1 above)
Comment (baselakasha) [reply index.js]: Sure, done.

**Comment by baselakasha** [comment]
@srt32 Thanks for the merge. Just a reminder that I did not update the example and the inputs in the README file.

**Comment by srt32** [comment]
Thanks! I'd happily take another docs PR :)

I can follow up later if not.

**Pull request merged**